src/components/Autocomplete.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Autocomplete = ({ suggestions, onClick, selectedIndex }) => {
+  if (!suggestions) return null;
```

Comment:
[very minor/code style] It's usually preferred to add an empty line after a guard clause such as this. Up to you, though.